zebra-network/src/protocol/external/message.rs

```diff
@@ -340,20 +340,29 @@ pub enum Message {
     FilterClear,
 }
 
+/// The maximum size of the rejection message
+const MAX_REJECT_MESSAGE_LENGTH: usize = 12;
```

Comment:
From https://github.com/zcash/zcash/blob/1b5ab4a06e52c23aca48afb1ebb72a25ef5dd3ca/src/main.cpp#L7334-L7335 we know zcashd uses `COMMAND_SIZE` for the message field. This is equal to 12:  https://github.com/zcash/zcash/blob/1b5ab4a06e52c23aca48afb1ebb72a25ef5dd3ca/src/protocol.h#L33

Reply:
Can you put this explanation in a doc comment?

It will help us understand why we chose `12` when we look at this code later.